gnome-extra/gnome-commander/gnome-commander-1.18.3.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome2 meson optfeature virtualx
+
+DESCRIPTION="A graphical, full featured, twin-panel file manager"
+HOMEPAGE="https://gcmd.github.io/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc exif gsf pdf samba taglib test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
```

Comment:
Running:
pkgdiff-mg --build-system gnome-commander-1.18.1-r1.ebuild gnome-commander-1.18.3.ebuild

I can see that a dependency on gdk-pixbuf was added:
+pixbuf    = dependency('gdk-pixbuf-2.0')
 
 common_deps = [
   glib, gmodule, gio, gobject, gtk
 ]
 
+# Require that gdk-pixbuf can load xpm files
+pixbuf_module_dir = pixbuf.get_variable(pkgconfig: 'gdk_pixbuf_moduledir')
+cpp.find_library('pixbufloader-xpm', dirs: pixbuf_module_dir, required: true)
+

Reply:
This has been fixed.